apps/src/music/redux/musicRedux.ts

```diff
@@ -191,6 +202,10 @@ const musicSlice = createSlice({
     ) => {
       state.undoStatus = action.payload;
     },
+    setShowCallout: (state, action: PayloadAction<string>) => {
```

Comment:
nit: personal preference, but I'd name the redux action something more action oriented like `showCallout` and the state value something like `currentCallout` or `showingCallout` - that way the meaning is more clear when you see something like `dispatch(showCallout('some-callout-id'))`